nbs/animation.ipynb

```diff
@@ -197,6 +198,23 @@
         "[args_d.update(a.param.values()) for a in arg_objs]\n",
         "args=AnimationArgs(**args_d)\n",
         "\n",
+        "\n",
+        "# load override settings if provided\n",
+        "if len(override_settings_path):\n",
```

Comment:
this is fine, but just for future reference I'm pretty sure an empty string evaluates to false in boolean statements. i.e. this could be simplified to `if override_settings_path:`

Reply:
Ah yes will change 👍 Verified this is indeed the case